pentaho-spring-dm-extender/pom.xml

```diff
@@ -17,22 +17,34 @@
     <exam.version>4.1.0</exam.version>
   </properties>
   <dependencies>
-    <dependency>
-      <groupId>org.springframework</groupId>
-      <artifactId>spring-beans</artifactId>
-    </dependency>
     <dependency>
       <groupId>org.springframework</groupId>
       <artifactId>spring-core</artifactId>
     </dependency>
+
+    <!-- region - Needed for compilation -->
```

Comment:
I don't understand this region. Not all compile dependencies are in it.

Reply:
I am reviewing it